locales/en/faq/faq55.md

```diff
@@ -0,0 +1 @@
+IO è stato concepito e sviluppato secondo i principi e le regole imposte dalla normativa sulla protezione dei dati personali, ovvero il nuovo Regolamento (UE) 2016/679 (cosiddetto “GDPR”) e il Codice privacy italiano (d. lgs. 196/2003, come recentemente emendato); esso è quindi uno strumento sicuro che protegge le tue informazioni e i tuoi diritti. IO però rappresenta solo uno strumento che le Pubbliche Amministrazioni utilizzano per interfacciarsi con i cittadini, quindi quest’ultime restano titolari dei tuoi dati personali, esattamente come avviene oggi, anche se per i trattamenti effettuati tramite l’app, in questa fase di sperimentazione, il Team Digitale è coinvolto in qualità di contitolare. Il Team Digitale sta dialogando con il Garante per la protezione dei dati personali per rendere IO uno strumento che migliori l’esperienza del cittadino anche sotto il profilo di accessibilità delle informazioni relative alla sua privacy. Quando IO diventerà il punto di accesso telematico tra cittadini e pubbliche amministrazioni, le operazioni di trattamento effettuate tramite IO saranno sottoposte al vaglio del Garante, secondo la procedura indicata all’articolo 71 del CAD (Codice dell’Amministrazione Digitale).
```

Comment:
```suggestion
IO è stato concepito e sviluppato secondo i principi e le regole imposte dalla normativa sulla protezione dei dati personali, ovvero il nuovo [Regolamento (UE) 2016/679 (cosiddetto “GDPR”)](https://eur-lex.europa.eu/legal-content/IT/TXT/HTML/?uri=CELEX:32016R0679&from=EN) e il [Codice privacy italiano](https://www.normattiva.it/uri-res/N2Ls?urn:nir:stato:decreto.legislativo:2003-06-30;196!vig=) (d. lgs. 196/2003, come recentemente emendato); esso è quindi uno strumento sicuro che protegge le tue informazioni e i tuoi diritti. IO però rappresenta solo uno canale che le Pubbliche Amministrazioni utilizzano per interfacciarsi con i cittadini, quindi queste ultime restano titolari dei tuoi dati personali, esattamente come avviene oggi. Il team di sviluppo di IO sta dialogando con il [Garante per la protezione dei dati personali](https://www.garanteprivacy.it/) per rendere IO uno strumento che migliori l’esperienza del cittadino anche sotto il profilo di accessibilità e trasparenza delle informazioni relative alla sua privacy. Quando IO diventerà il punto di accesso telematico tra cittadini e pubbliche amministrazioni, le operazioni di trattamento effettuate tramite IO saranno sottoposte al vaglio del Garante, secondo la procedura indicata all’art. 71 del [CAD (Codice dell’Amministrazione Digitale)](https://docs.italia.it/italia/piano-triennale-ict/codice-amministrazione-digitale-docs/it/v2017-12-13/_rst/capo5_sezione3_art64-bis.html)
```
We have update it just right now

Reply:
updated within https://github.com/pagopa/io-app/pull/1640/commits/b086c072b7637e3fd68d1a856bcd5568f8b6052d.